waypoint_saver_node:
  ros__parameters:
    saving_waypoints_file_name: 'shulab.csv'
    
    use_keyboard: False

    use_joy: True
    save_button: 0
    quit_button: 1
    erace_button: 2

    auto_record: false
    waypoint_interval: 4.0